src/Forms/FieldList.php

```diff
@@ -407,6 +407,9 @@ public function findTab($tabName)
         $currentPointer = $this;
 
         foreach ($parts as $k => $part) {
+            if ($currentPointer === null) {
+                return null;
+            }
```

Comment:
This isn't strictly needed, but it makes the test easier.
It seemed silly to be throwing an exception when `$currentPointer` is null, because in this scenario the tab doesn't exist so we can just return `null` from the method.